.github/workflows/unit-tests.yml

```diff
@@ -24,6 +24,19 @@ jobs:
         uses: ./.github/actions/init-environment 
       - name: Run unit tests
         run: make test/unit
+  test-bare:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9"]
+    steps:
+      - name: Checkout actions
+        uses: actions/checkout@v3
+      - name: Init environment 
+        uses: ./.github/actions/init-bare-environment 
+      - name: Run unit tests
+        run: poetry run pytest -n auto tests/unit/structures
```

Comment:
That's all the tests we can run? I suppose that is sufficient to cover the majority of missed optional imports, right?